shelley/chain-and-ledger/formal-spec/properties.tex

```diff
@@ -66,6 +66,14 @@ \subsection{Validity of a Ledger State}
 from the genesis state via valid LEDGER STS transitions. This gives a
 constructive rule how to reach a valid ledger state.
 
+
+\begin{property}[\textbf{Genesis Keys are Always All Delegated}]
+  \label{prop:genkeys-delegated}
+
+  All of the genesis keys are always all delegated.
```

Comment:
This is a great property! In order to test it, we should probably rephrase the property as something like "the size of the genesis key delegation mapping never changes". Since, from the ledger's point of view, "all of the genesis keys" is just the domain of this mapping.